src/lib/support/ZclString.h

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include <support/Span.h>
+
+#pragma once
+namespace {
+/**
+ * @brief Create ZCL string from char
+ */
+static CHIP_ERROR MakeZclCharString(chip::MutableByteSpan & buffer, const char * cString)
```

Comment:
Shouldn't need static inside anonymous namespace.

But also, shouldn't we just have a .cpp file for the implementation?

Reply:
Updated.